### PR TITLE
Layout user page (index, category, product, login, register)

### PR DESCRIPTION
**Pull request by mvn-vyle-dn**
## Description
- Layout user page (index, category, product, login, register)
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -28,3 +28,9 @@
     Route::resource('products', 'ProductController');
     Route::resource('orders', 'OrderController');
 });
+
+Route::group(['namespace' => 'Home'], function () {
```
Comment (at-quangtran) [inline routes/web.php]: api thì bỏ vào file api.php chứ

File: routes/web.php [hunk 2]
```diff
@@ -28,3 +28,9 @@
...
+    Route::get('/', 'HomeController@index')->name('user.home');
+    Route::get('/danh-muc', 'CategoryController@index')->name('user.category');
```
Comment (at-quangtran) [inline routes/web.php]: đổi sang tiếng anh

File: routes/web.php [hunk 3]
```diff
@@ -28,3 +28,9 @@
     Route::resource('products', 'ProductController');
     Route::resource('orders', 'OrderController');
 });
+
+Route::group(['namespace' => 'Home'], function () {
+    Route::get('/', 'HomeController@index')->name('user.home');
+    Route::get('/category', 'CategoryController@index')->name('user.category');
```
Comment (at-quangtran) [inline routes/web.php]: categories

File: routes/web.php [hunk 4]
```diff
@@ -28,3 +28,9 @@
...
+    Route::get('/product', 'ProductController@index')->name('user.product');
```
Comment (at-quangtran) [inline routes/web.php]: products

**Review by mvn-quangtran-dn — APPROVED** [review]
LGTM

**Pull request merged**